.eslintrc

```diff
@@ -7,8 +7,45 @@
     },
   },
   "rules" : {
-    "semi": [2, "always"],
-    "strict": [2, "global"]
+    "valid-jsdoc": [1],
```

Comment:
correct. Once all of them are in, these go away and the external link goes live.